### PR TITLE
Update the demo build to use the new tokens

### PR DESCRIPTION
**Pull request by hjheath**
Allow full integration tests of uploading files.

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://173-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by hjheath** [comment]
The travis link above should allow end to end testing of file upload.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://173-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://173-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://173-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://173-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by stefaniacardenas** [comment]
Looks good! Please rebase before merging

File: webpack.config.babel.js [hunk 1]
```diff
@@ -31,11 +32,31 @@ const styleBaseLoaders = [
   `less?sourceMap=${CSS_MAPS}`
 ];
 
+const PROD_CONFIG = {
```
Comment (stefaniacardenas) [inline webpack.config.babel.js]: Maybe we should have a separate config file where we store the configurations if these constants increase

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-173-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-173-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-173-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://173-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-173-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://173-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by hjheath** [comment]
Please could you review @rfreitas @stefaniacardenas 

We now build staging and production in parallel on travis and cache the npm packages. The down side is that we end up with two comments on the PR, but I think it's worth the speed up.

File: src/index.ejs [hunk 2]
```diff
@@ -46,49 +43,49 @@
...
+              type:'document',
+              options:{
+                useWebcam: options.useWebcam === "true"
+              }
+            },
+            'face',
+            'complete'
+          ]
+        })
+        var url = '<%= htmlWebpackPlugin.options.JWT_FACTORY %>'
```
Comment (hjheath) [inline src/index.ejs]: I think using the Define plugin in html files stopped working in webpack 2.0.

This workaround comes from:
https://github.com/webpack/webpack/issues/3105
https://stackoverflow.com/questions/39902197/how-can-i-pass-webpack-environment-variables-in-html

File: webpack.config.babel.js [hunk 3]
```diff
@@ -60,20 +68,29 @@ const baseStyleLoaders = [
...
 const basePlugins = [
   new webpack.NoEmitOnErrorsPlugin(),
   new webpack.DefinePlugin({
-    'process.env': {
-      'NODE_ENV': JSON.stringify(ENV),
-      'ONFIDO_API_URL': JSON.stringify(ONFIDO_API_URL),
-      'ONFIDO_SDK_SERVER_URL': JSON.stringify(ONFIDO_SDK_SERVER_URL),
-      'SDK_VERSION': JSON.stringify(packageJson.version)
-    }
+    'process.env.NODE_ENV': JSON.stringify(WEBPACK_ENV),
```
Comment (hjheath) [inline webpack.config.babel.js]: @rfreitas Do you know why we have to set this? I could not work out which dependency relies on it.

Comment (rfreitas) [inline webpack.config.babel.js]: probably none. remove it and see if it still works.

File: src/index.ejs (hunk 2 above)
Comment (rfreitas) [inline src/index.ejs]: I didn't even know it was possible to do it inside the html.

File: webpack.config.babel.js [hunk 4]
```diff
@@ -6,7 +6,15 @@ import autoprefixer from 'autoprefixer';
 import customMedia from 'postcss-custom-media';
 import url from 'postcss-url';
 
-const ENV = process.env.NODE_ENV || 'development';
+// ENV can be one of: development | staging | production
```
Comment (rfreitas) [inline webpack.config.babel.js]: 👍

File: src/index.ejs (hunk 2 above)
Comment (rfreitas) [inline src/index.ejs]: I see... you made it a template.

Comment (rfreitas) [inline src/index.ejs]: the other alternative is to use chunks! But this works, so we can roll with it.

**Comment by rfreitas** [comment]
LGTM 👍

File: webpack.config.babel.js (hunk 3 above)
Comment (hjheath) [inline webpack.config.babel.js]: It does not work, but I could not work out why we need it from the error.

**Comment by stefaniacardenas** [comment]
Looks good to me 👍

**Pull request merged**